skbio/io/format/stockholm.py

```diff
@@ -577,3 +576,79 @@ def _check_for_malformed_line(line, expected_len):
         raise StockholmFormatError('Line contains %d item(s). It must '
                                    'contain exactly %d item(s).'
                                    % (len(line), expected_len))
+
+
+@stockholm.writer(TabularMSA)
+def _tabular_msa_to_stockholm(obj, fh):
+    index = list(obj.index)
```

Comment:
Converting the index to a list shouldn't be necessary, it's already pretty list-like